cypress/support/step_definitions/timetable.js

```diff
@@ -1,10 +1,57 @@
-import { And, Then } from 'cypress-cucumber-preprocessor/steps';
+import { And, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { activity, appointment } from '../../mockApis/data';
+
+import { format, addDays } from 'date-fns';
 
 Then('I am displayed todays timetable', () => {
   cy.get('.todays-events > .govuk-body').contains(/.+/);
   cy.log('Timetable data has been passed through');
 });
 
-And('click the {string}', timetableLink => {
-  cy.get('a').contains(timetableLink).first().click();
+const daysFromNow = n => format(addDays(new Date(), n), 'yyyy-MM-dd');
+
+const tableToObject = table => {
```

Comment:
this could be something used across step definitions

Tables are passed through in this structure:
```
[ 
 ["header1", "header2" ],
 ["a1", "a2" ],
 ["b1", "b2" ],
]
```

and this will convert the table to this structure:
```
[
{"header1": "a1", "header2": "a2"},
{"header1": "b1", "header2": "b2"},
]
```